src/style.css

```diff
@@ -1,3 +1,90 @@
+html {
+  --active-link-color: #00acdc;
+  --menu-item-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation {
+  margin-right: 50px;
+}
+
+.menu-list {
+  display: flex;
+}
+
+.menu-list__item {
+  line-height: var(--menu-item-height);
+  display: flex;
+  height: var(--menu-item-height);
+}
+
+.menu-list__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.menu-list__link {
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: rgb(0, 0, 0);
```

Comment:
Use `hex` instead

Reply:
This is not fixed yet, by `hex` @BudnikOleksii meant for you to use `#000000` as color here.